packages/lab/src/pagination/Pagination.tsx

```diff
@@ -16,27 +16,37 @@ import { withBaseName } from "./utils";
 const { Provider } = paginationContext;
 
 export interface PaginationProps extends HTMLAttributes<HTMLElement> {
+  /**
+   * Number of pages.
+   */
   count: number;
+  /**
+   * Current/active page.
+   */
   page?: number;
+  /**
+   * Default current/active page.
+   */
   initialPage?: number;
```

Comment:
We've moved away from the `intial` wording and we use `default` in all of our components